audit-triplestore-web/src/main/resources/application.properties

```diff
@@ -3,3 +3,4 @@ jms.fcrepoEndpoint=${fcrepo.jms.endpoint:topic:fedora}
 jms.brokerUrl=failover:(tcp://${fcrepo.jms.host:localhost}:${fcrepo.dynamic.jms.port:61616})?startupMaxReconnectAttempts=1&initialReconnectDelay=1
 triplestore.baseUrl=${fcrepo.audit.triplestore.baseUrl:localhost:3030/test/update}
 event.baseUri=${fcrepo.audit.baseUri:http://example.com/event}
+audit.container=${fcrepo.audit.container:/audit}
```

Comment:
I dithered about this -- but half of the events are about /audit itself.  I'll try doing a check for startsWith(/audit/) or equals(/audit).